torchaudio/csrc/ffmpeg/stream_writer/stream_writer.h

```diff
@@ -70,11 +139,26 @@ class StreamWriter {
   // Write methods
   //////////////////////////////////////////////////////////////////////////////
  public:
+  /// Open the output file / device and write the header.
+  ///
+  /// @param opt Private options for protocol, device and muxer.
   void open(const c10::optional<OptionDict>& opt);
+  /// Close the output file / device and finalize metadata.
   void close();
 
+  /// Write audio data
+  /// @param i Stream index.
+  /// @param chunk Waveform tensor. Shape: ``(frame, channel)``.
+  /// The ``dtype`` must match what was passed to ``add_audio_stream()`` method.
   void write_audio_chunk(int i, const torch::Tensor& chunk);
+  /// Write video data
+  /// @param i Stream index.
+  /// @param chunk Video/image tensor. Shape: ``(time, channel, height,
+  /// width)``. The ``dtype`` must be ``torch.uint8``. The shape ``(height,
+  /// width and the number of channels)`` must match what was configured when
+  /// calling ``add_video_stream()``.
   void write_video_chunk(int i, const torch::Tensor& chunk);
+  /// Flush the frames from encoders and write the frames to the destination.
```

Comment:
should it be encoder?
```suggestion
  /// Flush the frames from encoder and write the frames to the destination.
```

Reply:
Nope, each stream has dedicated encoder. so there are multiple of them.